yarn-project/archiver/src/archiver/archiver_store_test_suite.ts

```diff
@@ -369,8 +369,6 @@ export function describeArchiverDataStore(testName: string, getStore: () => Arch
         await expect(store.getContractData(block.body.txEffects[0].contractData[0].contractAddress)).resolves.toEqual(
           block.body.txEffects[0].contractData[0],
         );
-
-        expect(block.body.txEffects[0].contractData[1]).toBe(undefined);
```

Comment:
When I changed the type to tuple it failed to compile because the compiler determined that index 1 does not fit into the tuple (since length is currently 1). So this test is now unnecessary because we have a compile time check.